kcidb/test_oo.py

```diff
@@ -0,0 +1,960 @@
+
+        if not self.source.is_initialized():
+            self.source.init()
+
+        self.source.load(
+            {
+                "version": {
+                    "major": 3,
+                    "minor": 0
+                },
+                "revisions": [
+                    {
+                        "id": "82bcf49e5e6ad570ff61ffcd210cf85c5ec8d896",
+                        "origin": "kernelci",
+                    }
+                ],
+                "builds": [
+                    {
+                        "revision_id":
+                            "82bcf49e5e6ad570ff61ffcd210cf85c5ec8d896",
+                        "id": "kernelci:kernelci.org:"
+                              "619c64b1712847eccbf2efac",
+                        "origin": "kernelci"
+                    }
+                ],
+                "tests": [
+                    {
+                        "waived": False,
+                        "start_time": "2021-11-23T03:52:13.660000+00:00",
+                        "path": "baseline.login",
+                        "status": "PASS",
+                        "id": "kernelci:kernelci.org:619c656de1fb4af479f2efaa",
+                        "build_id": "kernelci:kernelci.org"
+                                    ":619c64b1712847eccbf2efac",
+                        "origin": "kernelci"
+                    },
+                    {
+                        "waived": False,
+                        "start_time": "2021-11-23T03:52:13.666000+00:00",
+                        "path": "baseline.dmesg.crit",
+                        "status": "PASS",
+                        "id": "kernelci:kernelci.org:619c656de1fb4af479f2efac",
+                        "build_id": "kernelci:kernelci.org"
+                                    ":619c64b1712847eccbf2efac",
+                        "origin": "kernelci"
+                    },
+                    {
+                        "waived": False,
+                        "start_time": "2021-11-23T03:52:13.668000+00:00",
+                        "path": "baseline.dmesg.alert",
+                        "status": "PASS",
+                        "id": "kernelci:kernelci.org:619c656de1fb4af479f2efad",
+                        "build_id":
+                            "kernelci:kernelci.org:619c64b1712847eccbf2efac",
+                        "origin": "kernelci"
+                    },
+                    {
+                        "waived": False,
+                        "start_time": "2021-11-23T03:52:13.671000+00:00",
+                        "path": "baseline.dmesg.emerg",
+                        "status": "PASS",
+                        "id": "kernelci:kernelci.org:619c656de1fb4af479f2efae",
+                        "build_id":
+                            "kernelci:kernelci.org:619c64b1712847eccbf2efac",
+                        "origin": "kernelci"
+                    }
+                ]
+            }
+        )
+
+        self.source.load(
```

Comment:
Any particular reason why you are loading the data in two chunks and not just one?